tests/unit/NavigationModelTest.php

```diff
@@ -142,37 +138,51 @@ function testParseLines() {
 	 * @slowExecutionTime 0.02663 ms
 	 */
 	function testParseMessage() {
-		$messageName = 'test'.rand();
+		$messageName = 'test' . rand();
 
-		$this->getGlobalFunctionMock( 'wfMsg' )
-			->expects( $this->exactly( 2 ) )
-			->method( 'wfMsg' )
+		$messageMock = $this->getMock( 'stdClass', array( 'text', 'inContentLanguage' ) );
+		$messageMock->expects( $this->any() )
+			->method( 'text' )
 			->will( $this->returnValue( '*whatever' ) );
 
+		$messageMock->expects( $this->any() )
+			->method( 'inContentLanguage' )
+			->will( $this->returnValue( $messageMock ) );
+
+		$this->getGlobalFunctionMock( 'wfMessage' )
+			->expects( $this->exactly( 2 ) )
+			->method( 'wfMessage' )
+			->will(
+				$this->returnValue( $messageMock )
+			);
+
 		$model = new NavigationModel();
 
-		$nodes = array(
-			array(
-				'children' => array(1)
-			),
-			array (
+		$nodes = [
+			[
+				'children' => [ 1 ]
+			],
+			[
 				'original' => 'whatever',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "whatever" 14 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)